### DISCUSSION TITLE
吾爱破解 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#56] [模板讨论]
吾爱破解 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by jerrell851** [comment]
之前都是好的，用了很久，今天突然签不了啦，提示错误如下：

<html>
<body>


时间 | 状态 | 日志
-- | -- | --
2022-5-27 15:04:48 | 失败 | Failed at 4/8 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Request URL: {{checkUrl}}
2022-5-27 14:47:05 | 失败 | Failed at 4/8 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Response Error : OK,Request URL: {{checkUrl}}
2022-5-27 14:41:58 | 失败 | Failed at 4/8 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Response Error : OK,Request URL: {{checkUrl}}
2022-5-27 14:31:57 | 失败 | Failed at 2/8 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 302: Found,Request URL: https://www.52pojie.cn/home.php?mod=task&do=apply&id=2


</body>
</html>

**Reply by wjf0214** [reply]
更新一下看看，刚调整了下。好像是验证去掉了。

**Comment by xlon72** [comment]
一样的情况，第4步错误，模板编辑下，手动一次点击可以。自动运行就错误
Failed at 4/8 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Response Error : OK,
Request URL: {{checkUrl}}

**Reply by wjf0214** [reply]
更新一下看看，刚调整了下。好像是验证去掉了。

**Comment by zadishu** [comment]
显示成功，实际未打卡

**Reply by wjf0214** [reply]
我这依旧签到正常。看你cookie对不对。

**Reply by zadishu** [reply]
![WeChat截图_20230803134404](https://github.com/wjf0214/qd-templates/assets/110985414/4c4adc2b-1b4a-4bff-ba79-1200ea10ffe3)
 cookie 是对的

**Reply by wjf0214** [reply]
你这看日志，不是我模板啊。。。我的日志多一点。你更新看看？

**Reply by zadishu** [reply]
[](url)
![IMG_0567](https://github.com/wjf0214/qd-templates/assets/110985414/bbc06866-f9ea-4784-b928-abb64cae1b86)
s

**Reply by wjf0214** [reply]
我刚发现我在跑的和发布的不一样。可能之前调试忘了更新了。你再试试。

**Reply by zadishu** [reply]
用了最新的，还是这样

**Reply by zadishu** [reply]
![WeChat截图_20230808163146](https://github.com/wjf0214/qd-templates/assets/110985414/057bee3e-0729-4d9f-85ba-77f212eefdcb)
![WeChat截图_20230808163133](https://github.com/wjf0214/qd-templates/assets/110985414/8356e7af-e68f-4ec1-9194-6dee9ddaf0bf)

**Comment by shinesky** [comment]
Failed at 1/8 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.52pojie.cn/

**Reply by wjf0214** [reply]
第一条就403的话，看看IP是不是被ban了。

**Comment by xuexijiaoben** [comment]
Failed at 4/8 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Request URL: {{checkUrl}}

今天出这个问题了，换CK也 不行

**Reply by zhanbao2000** [reply]
同样的问题

**Reply by wjf0214** [reply]
估计是加了一些验证导致的，不过目前复现不出来。

**Reply by ZYMirror** [reply]
我也是遇到这样的问题

**Reply by dj1234618** [reply]
同样的问题

**Reply by icedog111** [reply]
现在还是不行，大佬可以解决吗

**Comment by tangb4c** [comment]
<img width="592" alt="image" src="https://github.com/wjf0214/qd-templates/assets/5152043/12310a4b-0d8b-41a0-a8ab-da9f6acdddfe">

为什么新建任务里，看不到cookie等字段？

**Comment by hecarli555** [comment]
这个签到是不是坏了？

**Reply by ZYMirror** [reply]
反正我用不了

**Comment by SayXieXie** [comment]
现在是不是不能用了？

**Reply by hecarli555** [reply]
早就坏了咯

**Comment by Melody-Test** [comment]
不再维护这个了吗大佬
